configure.ac

```diff
@@ -1118,7 +1118,8 @@ M0_CPPFLAGS_DIST=$(echo "$M0_CPPFLAGS" | grep -Po -e '-D\S+|-I/usr\S+|-inc\S+' |
 # __attribute__, such as __attribute__((gccxml(...))). It's important because we
 # use -Werror, which turns warnings into errors, but we still need to be able to
 # use gccxml attributes.
```

Comment:
Add a todo here once remove this flag once deprecated API's https://jts.seagate.com/browse/CORTX-34200 are fixed